out/build/x64-debug/VSInheritEnvironments.txt

```diff
@@ -0,0 +1 @@
+msvc_x64_x64
```

Comment:
aarg why does this keep happening.

Reply:
is out/build something used by default by your IDE? If so you can make a PR to add this to the .gitignore file
if not, consider using build/ instead since that is already in the repo's .gitignore